tests/args/specific_path.rs

```diff
@@ -82,6 +82,52 @@ fn checks_two_specific_files() {
     testdir.test_command_fail_with_args(args, expected_output);
 }
 
+#[test]
+fn checks_each_file_only_once_when_listing_same_path_twice() {
+    let testdir = TestDir::new();
+    testdir.create_testfile(".env", "foo=\n");
+
+    let subdir = testdir.subdir();
+    let testfile_1 = subdir.create_testfile(".env", " FOO=\n");
+    let testfile_2 = subdir.create_testfile(".env", " FOO=val\nBAR=foo\n");
```

Comment:
You can't create files with the same name in the same directory.

Reply:
It automatically created as env2. I can make it explicit though.